src/components/AudioPlayer/AudioPlayer.js

```diff
@@ -29,25 +34,46 @@ class AudioPlayerSub extends Component {
         TrackPlayer.CAPABILITY_PAUSE,
       ],
     })
-    const { track, playing, autoplay } = this.props
+
+    const { track, playing, autoplay, updatePlaying } = this.props
+
+    const id = uuid()
+
     // Adds the specified song to the track player to be ready to play
     await TrackPlayer.add({
       // Every track needs a unique id, not really used anywhere here though
-      id: uuid(),
+      id: id,
```

Comment:
```suggestion
      id,
```
You can use self-referencing here